src/zenml/artifacts/external_artifact_config.py

```diff
@@ -167,7 +167,7 @@ def get_artifact_id(
         client = Client()
 
         if self.id:
-            response = client.get_artifact(artifact_id=self.id)
+            response = client.get_artifact(self.id)
```

Comment:
Why remove the attribute name?

Reply:
Since it changed :P

Reply:
Maybe better to use new name then?

Reply:
don't really think we need arg names. here since there's only a single arg provided and it's obvious what it is used for